### ISSUE TITLE
"u:[username]" or "user:[username]" not working on 0.7.28, testen on linux and win servers, ip:[ip] works great, but we need username filter too )

### ISSUE THREAD

**Issue by angelexe** [Ylianst/MeshCentral#2092]
"u:[username]" or "user:[username]" not working on 0.7.28, testen on linux and win servers, ip:[ip] works great, but we need username filter too )

_Originally posted by @hellofaduck in https://github.com/Ylianst/MeshCentral/issues/1071#issuecomment-747963190_

**Comment by Ylianst** [comment]
When you go to list view, do you see the users show up correctly?

![image](https://user-images.githubusercontent.com/1319013/102643945-e447e380-4114-11eb-897f-331d0f507771.png)

**Comment by angelexe** [comment]
Users are displayed correctly, but the search for them does not work.

**Comment by hellofaduck** [comment]
Usernames displayed correctly on my setup too. And one more problem, before i started to search how to filter computers by username through built-in filter i searching for it by Ctrl+F, but after update this functionality brokes, page seems rendering now part by part as i scroling through it. This is a bug o a new way to render page?

**Comment by Ylianst** [comment]
Yes, it's the new rendering system, I had to do that because when you have a lot of devices the web site would get very slow. I have not considered that "CTRL-F" would no longer work, indeed, that will not work now.

I did some testing and I don't see any issues. When I type "u:mini" or "user:mini" on my server with the screen above, it filters correctly. Can you pull up the developer console for your browser and see if there are any errors?

![image](https://user-images.githubusercontent.com/1319013/102702344-91eeeb80-4216-11eb-8cb5-b4de98d9efba.png)

Basically, unless I can make this problem occur on my system, there is nothing I can do. I need the steps to replicate this.

**Comment by hellofaduck** [comment]
There is no any errors in my firefox dev console. But i found strange behavior of filter...for example, when i type "ip:" i see all computers and can search through it, but when i type "u:" or "user:" it shows 0 machines, it looks like some regex ot smth like this not working or not recognizing this keyword. But! i found also when i type "\" in filter field i can see all computers like in situation when i typing "ip:". I dont know what it means but i wonder this can help to diagnose...

**Comment by hellofaduck** [comment]
Tested this problem not only on my Firefox but on Edge(Chromium) too. Same problem and there is no errors in console (
If i can do something to help you debug this problem please ask me and i help with pleasure. i.e i can give you an access to my homelab(web or ssh) installation of MeshCentral, so you can see problem with your own eyes )

**Comment by hellofaduck** [comment]
Solved it! Uhaa! Problem hiding in different language, We using russian translation so my username column is "Пользователь" not "Username"! When i typing in filter box "Пользователь:" or "П:" it works like expected.
